Description: fix number of skipped tests
 This patch applies on top of ac5e11f which skips all tests if STDIN is missing.
 Still in a chroot "-t STDIN" can pass and the check for /dev/tty can fail,
 and the we run into this problem.
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-07-12

--- a/t/02_terminal_functions.t
+++ b/t/02_terminal_functions.t
@@ -38,7 +38,7 @@
             }
         }
     };
-    skip( 'Because Term::ReadKey need at least a tty to be useful', 1 ) if $@;
+    skip( 'Because Term::ReadKey need at least a tty to be useful', 7 ) if $@;
     *IN = *IN;    # Make single-use warning go away
     $|  = 1;
     no strict "subs";
